src/agent/onefuzz-supervisor/src/coordinator.rs

```diff
@@ -233,7 +234,13 @@ impl Coordinator {
         request_type: RequestType<'a>,
     ) -> Result<Response> {
         let request = self.build_request(request_type.clone())?;
-        let mut response = self.client.execute(request).await?;
```

Comment:
since self.client isn't used anymore, it should probably get removed all together.

Reply:
it is still used to create the RequestBuilder see poll_commands_request on line 267

Reply:
ah, yeah I missed that.